src/jobQueue.ts

```diff
@@ -11,7 +11,11 @@ import type {
 const logger = getLogger(__filename);
 
 const CONCURRENT_JOB_COUNT = 5;
-const POLL_INTERVAL_MS = 1000;
+/** The poll interval is for scheduled (not on-demand) jobs combined with idle workers. See
+ * https://worker.graphile.org/docs/faq#if-we-have-jobs-that-are-scheduled-in-the-futurefailed-will-workers-continuously-poll-to-run-those-jobs-or-will-the-listennotify-mechanism-be-used-for-that
+ * Every poll interval times the number of workers, a SQL update statement runs.
+ */
+const POLL_INTERVAL_MS = 600_000;
```

Comment:
It took me a moment to translate this number into what it actually means -- what do you think of adding  `ONCE_PER_MINUTE_MS = 600_000` to our `time.ts` constants list?

Truly if that seems silly no problem, though if not that it might not hurt to add `// once per minute` to this line just for human legibility.

Reply:
(Marking this as resolved to hide my shame)

Reply:
Meh, it's too many numerals. #1722 would help.